config/default.json

```diff
@@ -8,7 +8,7 @@
   },
   "mongodb": "mongodb://localhost:27017/gamification",
   "rabbitmq": "localhost:5672",
-  "rabbitQueue": "my-queue",
-  "rabbitExchange": "my-exchange",
-  "rabbitRoutingKey": "my-key"
+  "rabbitQueue": "openhpi-schulcloud-queue",
```

Comment:
Would it not make more sense to create a specific `schul-cloud` config instead of changing the default config?

Reply:
In production, `production.json` is loaded. So I will reset the change sin this file and then production.json has to be changed.